worldedit/manipulations.lua

```diff
@@ -640,11 +640,8 @@ function worldedit.clear_objects(pos1, pos2)
 
 	local function should_delete(obj)
 		-- Avoid players and WorldEdit entities

```

Comment:
comment is now wrong

Reply:
i re-added the is_player check; something like that is needed to ensure the count of removed objects is correct. the comment seems accurate now.